ipie/qmc/calc.py

```diff
@@ -137,15 +140,33 @@ def get_driver(options: dict, comm: MPI.COMM_WORLD) -> AFQMC:
         )
         propagator = Propagator[type(hamiltonian)](params.timestep)
         propagator.build(hamiltonian, trial, walkers, mpi_handler)
-        afqmc = AFQMC(
-            system,
-            hamiltonian,
-            trial,
-            walkers,
-            propagator,
-            params,
-            verbose=(verbosity and comm.rank == 0),
+        free_projection = get_input_value(
```

Comment:
So I think we should keep this outside the main driver for the moment. the json input file way of running is sort of outdated at this point and I think we should just move to running things directly in python.

Reply:
moved to addons